packages/backend/src/open_payments/payment/outgoing/lifecycle.ts

```diff
@@ -42,95 +38,76 @@ export async function handleSending(
 
-  if (receiver.incomingAmount && receiver.receivedAmount) {
-    const maxAmountToDeliver =
-      receiver.incomingAmount.value - receiver.receivedAmount.value
-    if (maxAmountToDeliver < newAmountToDeliver) {
-      newAmountToDeliver = maxAmountToDeliver
-    }
-  }
-
-  if (newAmountToDeliver <= BigInt(0)) {
-    // Payment is already (unexpectedly) done. Maybe this is a retry and the previous attempt failed to save the state to Postgres. Or the incoming payment could have been paid by a totally different payment in the time since the quote.
+  if (maxReceiveAmount <= BigInt(0)) {
+    // Payment is already (unexpectedly) done.
+    // Maybe this is a retry and the previous attempt failed to save the state to Postgres. Or the incoming payment could have been paid by a totally different payment in the time since the quote.
     deps.logger.warn(
       {
-        newMaxSourceAmount,
-        newAmountToDeliver,
+        maxDebitAmount,
+        maxReceiveAmount,
         amountSent,
         receiver
       },
       'handleSending payment was already paid'
     )
     await handleCompleted(deps, payment)
     return
-  } else if (
-    newMaxSourceAmount <= BigInt(0) ||
-    newAmountToDeliver <= BigInt(0)
-  ) {
+  }
+
+  if (maxDebitAmount <= BigInt(0)) {
     // Similar to the above, but not recoverable (at least not without a re-quote).
-    // I'm not sure whether this case is actually reachable, but handling it here is clearer than passing ilp-pay bad parameters.
+    // I'm not sure whether this case is actually reachable, but handling it here is clearer than passing in bad parameters.
     deps.logger.error(
       {
-        newMaxSourceAmount,
-        newAmountToDeliver
+        maxDebitAmount,
+        maxReceiveAmount
       },
       'handleSending bad retry state'
     )
     throw LifecycleError.BadState
   }
 
-  const lowEstimatedExchangeRate = payment.quote.lowEstimatedExchangeRate
-  const highEstimatedExchangeRate = payment.quote.highEstimatedExchangeRate
-  const minExchangeRate = payment.quote.minExchangeRate
-  const quote: Pay.Quote = {
-    ...payment.quote,
-    paymentType: Pay.PaymentType.FixedDelivery,
-    // Adjust quoted amounts to paymentPointer for prior partial payment.
-    maxSourceAmount: newMaxSourceAmount,
-    minDeliveryAmount: newAmountToDeliver,
-    lowEstimatedExchangeRate,
-    highEstimatedExchangeRate,
-    minExchangeRate
-  }
-
-  const destination = receiver.toResolvedPayment()
-  const receipt = await Pay.pay({ plugin, destination, quote }).finally(() => {
-    return Pay.closeConnection(plugin, destination).catch((err) => {
-      // Ignore connection close failures, all of the money was delivered.
-      deps.logger.warn(
-        {
-          destination: destination.destinationAddress,
-          error: err.message
-        },
-        'close pay connection failed'
-      )
-    })
+  await deps.paymentMethodHandlerService.pay('ILP', {
```

Comment:
Instead of hardcoding `ILP`, should we not make that configurable right away?

Reply:
We need several changes to the quote model for this to not be hardcoded so I think this should be changed during the PR that adds support for open payments payment methods

Reply:
We could address this in our mojaloop payment method effort when we actually begin utilizing a second payment method. It's not strictly necessary for updating rafiki to use the latest open-payments so I'm concerned it would unnecessarily slow that effort down. I can include it if we want its just not clear to me what this would take.